docs/manual/docs/cmd/spo/site/site-remove.md

```diff
@@ -1,9 +1,17 @@
-# spo site classic remove
+# spo site remove
```

Comment:
Release notes and Comparison to PowerShell contain references to the old .md file which leads to errors while building docs